HW_5/task28.py

```diff
@@ -0,0 +1,18 @@
+# 28  Напишите рекурсивную функцию sum(a, b), возвращающую сумму двух целых неотрицательных чисел
+# Из всех арифметических операций допускаются только +1 и -1.
+#  Также нельзя использовать циклы.
+# 2 2
+# 4'''
+
+a = int(input("Введите первое неотрицительное число "))
+b = int(input("Введите второе неотрицательно число "))
+
+
+def recursive_sum(a, b):
+    if a == 0:
+        return b
+    else:
+        return recursive_sum(a - 1, b + 1)
```

Comment:
Блок кода `else` можно не указывать.
```suggestion

    return recursive_sum(a - 1, b + 1)
```